### ISSUE TITLE
Map.contains(Only)EntriesOf

### ISSUE THREAD

**Issue by robstoll** [robstoll/atrium#173]
*Platform* (jvm, jdk8, js, android): 

## Code related feature
```
expect(map).containsEntriesOf(otherMap)
```
I suggest that we also introduce a builder if we add this:
```
expect(map).contains.inAnyOrder.entriesOf(...)
```
Which then should be extended with 
```
expect(map).contains.inAnyOrder.keyValuePair(Pair<..., ...>)
expect(map).contains.inAnyOrder.keyValuePairs(Pair<..., ...>, vararg Pair<..., ...>)
expect(map).contains.inAnyOrder.keyValue(KeyValue<..., ...>)
expect(map).contains.inAnyOrder.keyValues(KeyValue<..., ...>, vararg <..., ...>)
```
And we could extend it further with (please in a second PR, relates to #68, see requirements for reporting there):
```
expect(map).contains.inAnyOrder.only.entriesOf(Map<..., ...>)
expect(map).contains.inAnyOrder.only.keyValuePair(Pair<..., ...>)
expect(map).contains.inAnyOrder.only.keyValuePairs(Pair<..., ...>, vararg Pair<..., ...>)
expect(map).contains.inAnyOrder.only.keyValue(KeyValue<..., ...>)
expect(map).contains.inAnyOrder.only.keyValues(KeyValue<..., ...>, vararg <..., ...>)

expect(map).contains.inOrder.only.entriesOf(Map<..., ...>)
expect(map).contains.inOrder.only.keyValuePair(Pair<..., ...>)
expect(map).contains.inOrder.only.keyValuePairs(Pair<..., ...>, vararg Pair<..., ...>)
expect(map).contains.inOrder.only.keyValue(KeyValue<..., ...>)
expect(map).contains.inOrder.only.keyValues(KeyValue<..., ...>, vararg <..., ...>)
```

Please react with :+1: if you want to see `containsEntriesOf` implemented in Atrium, the more upvotes the more likely I will implement it myself -- feel free to [sponsor me](https://github.com/sponsors/robstoll), that would be a motivation too. 
You are of course welcome to work on this issue. Write `I'll work on it` as comment so that we can assign the task to you.
And if you have already a use case in mind, then it would be great if you share it via comment.

**Comment by piyushmor** [comment]
👍

**Comment by robstoll** [comment]
@piyushmor thanks for your feedback, I deleted your comment as I don't want to encourage people writing a comment with a single 👍 (otherwise this issue gets cluttered). I prefer the reaction.
Btw. if you know other people wanting this feature then encourage them to react with 👍 as well.

**Comment by robstoll** [comment]
I'll also add a shortcut for `contains o inAny order but only entriesOf mapLike` named `containsOnlyEntriesOf`

**Comment by robstoll** [comment]
@piyushmor, @stragasevera and @igorakkerman, pinging you guys because you voted for the issue. It is now available in Atrium 0.15.0 please let us know in case you miss something about this feature
